server/api/controllers/praises/get-praises.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+  friendlyName: 'Get praises',
+
+  description: 'Get all praises',
+
+  inputs: {},
```

Comment:
i think you should put the category as an input here to make it clear. cos you put the category in the fn but not in the input

Reply:
added category to inputs for clarification